### ISSUE TITLE
Reducing clicks

### ISSUE THREAD

**Issue by fonsuiplaysvideogames** [pyfa-org/Pyfa#2138]
I am opening an issue to discuss the subject of quantity of clicks required to accomplish a goal in general, and will provide a few specific suggestions. If that is too wide a scope for a single issue, individual ones can be created to determine their efficacy and if applicable track their progress.

In general, I come across UI functions that seem to be buried somewhere, when they could be brought closer to the "surface", i.e. requiring fewer clicks/menus to access. This leads to an unnecessarily onerous user experience.

A few specific examples:
1) This can be three buttons, or a radio select, instead of a menu that requires a click to access, and click to select an option:
![image](https://user-images.githubusercontent.com/60384561/74817425-e231b000-52ca-11ea-94af-2620695987dd.png)

2) Projected weather effects can be a radio select panel as well; I often find myself flipping between different weather projections or levels of projected weather effects:
![image](https://user-images.githubusercontent.com/60384561/74817616-3046b380-52cb-11ea-9eaa-566df515f7ff.png)

3) Standardized implant sets (a set containing all six slot 1-6 implants of each type and grade) added to this list would be very useful:
![image](https://user-images.githubusercontent.com/60384561/74817751-71d75e80-52cb-11ea-9865-64193b8e36fc.png)

4) _Adding_ a submenu to this context menu for "fill with [quantity x]" with options for 1-8 would reduce the number of clicks to get the precise correct number of modules fitted:
![image](https://user-images.githubusercontent.com/60384561/74817917-bbc04480-52cb-11ea-867a-b95b74788c80.png)

5) The "Change ship/module/charge skills" submenu can have a menu item added to the top which opens a new window allowing you to tweak all skills in the menu, similar to the mutaplasmid attributes interface:
![image](https://user-images.githubusercontent.com/60384561/74818209-38ebb980-52cc-11ea-8053-8d288e204758.png)

6) While not strictly a "clicks" related thing, this is UX based and relates to mouse-over and visible information - there is a lot of information displayed persistently, and a lot displayed by mouseover tooltip. Whether a piece of data should be listed on a tooltip or persistently in the main window, and precisely where that data should be located, can be user-configurable. Example of such data below:
![image](https://user-images.githubusercontent.com/60384561/74818561-dfd05580-52cc-11ea-8ef9-bc25e4f29b99.png)

![image](https://user-images.githubusercontent.com/60384561/74818616-fb3b6080-52cc-11ea-94b0-b988c124ffe4.png)
(Those are two separate tooltips for the same section/closely related data)
![image](https://user-images.githubusercontent.com/60384561/74818695-18702f00-52cd-11ea-96fd-ffa70fbb943f.png)

7) Factoring in reload time can be a single-click checkbox rather than a two-click menu:
![image](https://user-images.githubusercontent.com/60384561/74818786-45bcdd00-52cd-11ea-80fa-b0ab25308a43.png)

I feel these examples largely cover the point(s) I am attempting to make. Please let me know your thoughts, if any of these can/should be acted upon, and if so, if separate issues should be utilized to track them.

**Comment by DarkFenX** [comment]
I agree with most of your points. However, it's all :effort: for all points but 2 (tooltips and reload) if you want to make them look good. If anyone wants to pick those low-hanging fruits up - I will gladly accept the as pull requests.

Otherwise, looks like a good list of stuff to have in the Pyfa rewrite :P

**Comment by DarkFenX** [comment]
Added 3) in 2.19.0
